src/eigsolve/eigsolve.jl

```diff
@@ -292,3 +293,15 @@ function eigsort(which::Symbol)
     end
     return by, rev
 end
+
+function conjwhich(which::Symbol)
+    if which == :LI
+        return :SI
+    elseif which == :SI
+        return :LI
+    else
+        return which
+    end
+end
+
+conjwhich(s::EigSorter) = EigSorter(conj(s.by), s.rev)
```

Comment:
`s.by` can be a function, so I don't see how that would work? Do you mean something like `x->s.by(conj(x))` ?

Reply:
ah, my mistake, I only  tested with ```:LM``` so this escaped from my attention, should be fixed now